src/shape/oriented-box.js

```diff
@@ -19,17 +19,16 @@ var tmpMat4 = new Mat4();
  * @param {pc.Mat4} [worldTransform] - Transform that has the orientation and position of the box. Scale is assumed to be one.
  * @param {pc.Vec3} [halfExtents] - Half the distance across the box in each local axis. The constructor takes a reference of this parameter.
  */
-function OrientedBox(worldTransform, halfExtents) {
-    this.halfExtents = halfExtents || new Vec3(0.5, 0.5, 0.5);
+class OrientedBox {
+    constructor(worldTransform = new Mat4(), halfExtents = new Vec3(0.5, 0.5, 0.5)) {
+        this.halfExtents = halfExtents;
 
-    worldTransform = worldTransform || tmpMat4.setIdentity();
-    this._modelTransform = worldTransform.clone().invert();
+        this._modelTransform = worldTransform.clone().invert();
 
-    this._worldTransform = worldTransform.clone(); // temp - currently only used in the worldTransform accessor, see future PR for more use
-    this._aabb = new BoundingBox(new Vec3(), this.halfExtents);
-}
+        this._worldTransform = worldTransform.clone(); // temp - currently only used in the worldTransform accessor, see future PR for more use
```

Comment:
should probably remove the comment, seem obsolete / irrelevant

Reply:
Removed